test/suites/standard/version.robot

```diff
@@ -74,20 +75,27 @@ Teardown
     Logout MicroShift Host
 
 Read Expected Versions    # robocop: disable=too-many-calls-in-keyword
-    [Documentation]    Read ../Makefile.version.aarch64.var to find the expected versions
+    [Documentation]    Ask dnf for the version of the MicroShift package to
+    ...    find the expected versions
+    ...
     ...    Sets suite variables FULL_VERSION, MAJOR_VERSION, MINOR_VERSION, and Y_STREAM based on
     ...    the content.
 
-    # The file contains content in this format:
-    #
-    #    OCP_VERSION := 4.14.0-0.nightly-arm64-2023-05-04-012046
+    ${rc}=    SSHLibrary.Execute Command
+    ...    test -f /usr/bin/dnf
+    ...    return_rc=True
+    ...    return_stdout=False
+    ...    return_stderr=False
 
-    ${unparsed}=    OperatingSystem.Get File    ../Makefile.version.aarch64.var
-
-    # 4.14.0-0.nightly-arm64-2023-05-04-012046
-    ${version_full_raw}=    Fetch From Right    ${unparsed}    :=
-    ${version_full}=    Strip String    ${version_full_raw}
+    IF    ${rc}==0
+        # This returns a string like 4.14.0-0.nightly-arm64-2023-05-04-012046
+        ${unparsed}=    Get Version Of MicroShift RPM From DNF
+    ELSE
+        ${unparsed}=    Get Version Of MicroShift RPM From Ostree
+    END
```

Comment:
@dhellmann , since we are checking the version of an installed package, can we use `rpm -q` which works on both system types?

Reply:
Nice, I've updated it to work that way.